helpers/pathSupportsPlainText.js

```diff
@@ -0,0 +1,32 @@
+const contentType = "text/plain";
+
+const pathSupportsPlainText = (path) => {
+    if (path.requestBody) {
```

Comment:
Instead of a new pathSupportsPlainText.js file, how about changing the current pathSupportsJson.js -> pathSupports.js which has a method that takes in the type and the path as the methods look identical apart from the contentType string also keep the two different contentTypes as constants in pathSupports.js.